arch/cpu/cc26x0-cc13x0/rf-core/ieee-mode.c

```diff
@@ -1499,6 +1586,24 @@ set_object(radio_param_t param, const void *src, size_t size)
 
     return rv;
   }
+  if(param == RADIO_POWER_MODE_CARRIER_ON ||
+     param == RADIO_POWER_MODE_CARRIER_OFF) {
+    int channel = ((int *)src)[0];
```

Comment:
This is really ugly, can you at least have an array `int configuration[3]` like in the calling code? Better yet to have a `struct`.